src/ceph.in

```diff
@@ -853,7 +905,21 @@ def main():
         return 1
 
     if parsed_args.help:
-        return do_extended_help(parser, childargs)
+        return do_extended_help(parser, childargs, ('mon', ''), ' '.join(childargs))
+
+    # implement "tell service.id help"
+    if len(childargs) >= 3 and childargs[0] == 'tell' and childargs[2] == 'help':
+        target = childargs[1].split('.')
+        if validate_target(target):
+            if len(target) == 1: 
+                # TODO(Chang Liu): for "mon" target only, we should remove this
+                # patch after pybind supports this target format.
+                target.append("")
```

Comment:
I considered this question, the function should be simple, and only one function: to determine whether target is legal. so i put this fix out of the function.

I will try to fix it via `pybind` later. how about？